geode-pulse/src/main/webapp/WEB-INF/web.xml

```diff
@@ -32,7 +32,7 @@
   </servlet>
   <servlet-mapping>
     <servlet-name>mvc-dispatcher</servlet-name>
-    <url-pattern>/pulse/*</url-pattern>
```

Comment:
i see. if changing it back to /pulse/* makes no difference, then add it back for clarify.